<launch>

  <arg name="map" default="ManyObstaclesMap.yaml" />
  <node name="map_server" pkg="map_server" type="map_server" args="$(find bot_sim)/map/$(arg map)"/>

  <node pkg="bot_sim" type="nav_ctrl.py" name="nav_ctrl" output="screen" >
    <param name="world_frame" value="map"/>
    <param name="robot_frame" value="base_footprint"/>
    <param name="cost_map_path" value="/home/sentry_train_test/AstarTraining/sim_nav/src/bot_sim/scripts/DWA/CostMap/CostMapA2d5B1d8" />
    <param name="dyn_map_name" value="grid"/>
  </node>
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find bot_sim)/config/rviz/gazebo_laserbot.rviz" />

  <node pkg="bot_sim" type="RRT_star_srv.py" name="RRT_star_srv" />

  <node pkg="bot_sim" type="scan_2_node" name="scan_2_node">
    <param name="obstacle_enlargement" type="int" value="25" />
    <param name="MAXN" type="int" value="100" />
    <param name="threshold" type="int" value="25" />
    <param name="scan_topic" type="string" value="scan_laser" />
    <param name="obstacle_map_topic" type="string" value="grid" />
    <param name="base_frame" type="string" value="base_footprint" />
    <param name="laser_frame" type="string" value="laser_lrf_link" />
  </node>

</launch>
